### DISCUSSION TITLE
Is there a tree-like sidebar?

### DISCUSSION THREAD

**Discussion by kuchaguangjie** [saadeghi/daisyui#1504] [Q&A] [answered]
I've checked the Navbar, but seems it doesn't support multiple level (>=2) tree that can unfold / fold via clicking `+` and `-`.
So, is there a solution for that?
If u need a reference, it's something like this: https://flowbite-svelte.com/components/sidebar

**Comment by saadeghi** [comment, marked as answer]
This is something that needs to be added in the next major version. Because the changes will break the current projects.
But for now you can do something like this: https://github.com/saadeghi/daisyui/issues/1363 or you can make that functionality using some JS code.
